params/defaults.json

```diff
@@ -109,15 +109,15 @@
   },
   {
     "ParameterKey": "SolrCloudMaxSize",
-    "ParameterValue": "6"
+    "ParameterValue": "3"
   },
   {
-    "ParameterKey": "SolrCloudDockerImage",
-    "ParameterValue": "hybox/solr-ecs:6"
+    "ParameterKey": "SolrHealthReportingSystemType",
+    "ParameterValue": "enhanced"
   },
   {
-    "ParameterKey": "SolrCloudDataVolumePath",
-    "ParameterValue": "/var/lib/docker/vfs/dir/solr-data"
+    "ParameterKey": "SolrS3Key",
```

Comment:
What is actually stored in the Solr and Zookeeper S3 buckets? If these need to be created prior to building the stack, we'll need to add that to the readme. Also, can these two be the same bucket, or do they need to be distinct?